go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/deckarep/golang-set/v2 v2.1.0
 	github.com/dop251/goja v0.0.0-20230806174421-c933cf95e127
-	github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20240803025447-c92ef420eec2
+	github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20240806205917-f72450b810e2
```

Comment:
I assume the SCR reference is changing because it needs to point at a build familiar with Holocene?

Reply:
yep, that's the version that introduced the `HoloceneTime` field. But I may want to update the reference again, and there are other go.mod merge conflicts to resolve...